hw6.py

```diff
@@ -0,0 +1,98 @@
+class Exchanger(object):
+    _base = None
+
+    @staticmethod
+    def exchange(money, to_currency="USD"):
+        ratio = Exchanger._base.get((money.currency, to_currency))
+        if ratio is None:
+            # print("{} to {} ratio hasn't found! Used 1.0 as the ratio.")
+            ratio = 1.0
+        return Money(money.amount * ratio, to_currency)
+
+
+class ExchangerOnline(Exchanger):
+    @staticmethod
+    def exchange(money, to_currency="USD"):
+        if Exchanger._base is None:  # if dictionary is empty
+            data = ExchangerOnline.request_exchange_data()
+            Exchanger._base = ExchangerOnline.parse_exchange_data(data)
+        return Exchanger.exchange(money, to_currency)
+
+    @staticmethod
+    def request_exchange_data():
+        import json
```

Comment:
import statements have to be moved to the top of the file:[]( https://www.python.org/dev/peps/pep-0008/#imports)